.env.sample

```diff
@@ -247,3 +247,8 @@ CASSANDRA_DB = ""
 # OFFLINE TOKEN VALIDATION
 VALIDATE_ACCESS_TOKEN_OFFLINE = "ON/OFF"
 KEYCLOAK_PUBLIC_KEY_PATH = "keycloak-public-keys"
+
+IMPROVEMENT_FCM_KEY_PATH = /config/credentials/improvement-fcm-keystore.json
```

Comment:
@priyanka-TL - Move it under FCM configuration, change existing to DEFAULT_FCM_KEY_PATH